app/redux/reducers/misc.js

```diff
@@ -1140,3 +1140,14 @@ export const ssoEnabledDisplay = (state = initialState.ssoEnabledDisplay, action
       return state;
   }
 };
+
+export const pendoData = (state = initialState.pendoData, action) => {
+  switch (action.type) {
+    case types.SET_PENDO_DATA:
+      return action.payload.data;
+      case types.LOGOUT_REQUEST:
+        return initialState.pendoData;
```

Comment:
indentation seems shifted here
```suggestion
    case types.LOGOUT_REQUEST:
      return initialState.pendoData;
```